Unreal Launcher/Unreal Launcher/MainTab.xaml.cs

```diff
@@ -193,5 +209,13 @@ private void CheckBox_Log_Changed(object sender, RoutedEventArgs e)
 		{
 			Project.LaunchSettings.Log = CheckBox_Log.IsChecked ?? false;
 		}
+
+		private void ComboBox_SaveGame_SelectionChanged(object sender, SelectionChangedEventArgs e)

```

Comment:
We'd usually avoid using underscores in the middle of function names, though perhaps there's a specific reason for this one? As a callback, this would preferably start with "On"